libtransact/src/scheduler/serial/core.rs

```diff
@@ -57,54 +57,51 @@ pub enum CoreMessage {
 }
 
 #[derive(Debug)]
-#[allow(clippy::enum_variant_names)]
 enum CoreError {
-    ExecutionSendError(Box<SendError<ExecutionTask>>),
-    ContextManagerError(Box<ContextManagerError>),
-    InternalError(String),
+    ExecutionSend(Box<SendError<ExecutionTask>>),
+    ContextManager(Box<ContextManagerError>),
+    Internal(String),
 }
 
 impl std::error::Error for CoreError {
     fn description(&self) -> &str {
         match *self {
-            CoreError::ExecutionSendError(ref err) => err.description(),
-            CoreError::ContextManagerError(ref err) => err.description(),
-            CoreError::InternalError(ref err) => err,
+            CoreError::ExecutionSend(ref err) => err.description(),
+            CoreError::ContextManager(ref err) => err.description(),
+            CoreError::Internal(ref err) => err,
         }
     }
 
     fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
         match *self {
-            CoreError::ExecutionSendError(ref err) => Some(err),
-            CoreError::ContextManagerError(ref err) => Some(err),
-            CoreError::InternalError(_) => None,
+            CoreError::ExecutionSend(ref err) => Some(err),
+            CoreError::ContextManager(ref err) => Some(err),
+            CoreError::Internal(_) => None,
         }
     }
 }
 
 impl std::fmt::Display for CoreError {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match *self {
-            CoreError::ExecutionSendError(ref err) => {
-                write!(f, "ExecutionSendError: {}", err.description())
+            CoreError::ExecutionSend(ref err) => write!(f, "ExecutionSend: {}", err.description()),
```

Comment:
These should be more of a sentence.  For example:  `"Unable to send transaction for execution:"`

Reply:
Done